lib/cloud_foundry.rb

```diff
@@ -11,6 +11,15 @@ def self.vcap_data
     end
   end
 
+  def self.app_url
+    if self.is_environment?
+      # (arbitrarily) use the shortest assigned route, if there are multiple
```

Comment:
not strong feelings, I just feel like this comment says "this is arbitrary" so why not actually be arbitrary and use `take` to  get a random one?